packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1017,7 +1017,15 @@ abstract class Element<T extends Widget> implements BuildContext {
 }
 
 class ErrorWidget extends LeafRenderObjectWidget {
-  RenderBox createRenderObject() => new RenderErrorBox();
+  ErrorWidget(Object exception) : message = _stringify(exception), super(key: new GlobalKey());
```

Comment:
Oh, yeah, good call.

And yes, the idea here is to make sure we never sync with it. (In part because I intentionally don't implement the updateRenderObject protocol here, and the RenderErrorBox is immutable.)